pandas/core/groupby/groupby.py

```diff
@@ -2675,6 +2674,47 @@ def rank(
         Returns
         -------
         DataFrame with ranking of values within each group
+
+        See Also
+        --------
+        Series.groupby : Apply a function groupby to a Series.
+        DataFrame.groupby : Apply a function groupby
+            to each row or column of a DataFrame.
+        Series.rank : Apply a function rank to a Series.
+        DataFrame.rank : Apply a function rank
```

Comment:
Based on the screenshot you posted, looks like this doesn't render as a link, so not that useful in current form.  I think best to keep scope small and remove changes to the See Also (which could then be tackled as part of #42406 if you're interested!).

Reply:
reverted the change on `See Also`